clients/gax/mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule GoogleApi.Gax.MixProject do
   defp deps() do
     [
       {:tesla, "~> 1.0"},
-      {:poison, ">= 1.0.0"},
+      {:poison, "~> 3.1"},
```

Comment:
If you want to cast a wider net, you can say `">= 1.0.0 and < 4.0"`. See https://hexdocs.pm/elixir/Version.html#module-requirements